sdk/advisor/ci.yml

```diff
@@ -21,14 +35,11 @@ pr:
     include:
     - sdk/advisor/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
 
 jobs:
```

Comment:
This should probably be `stages`

Reply:
Whoops! Yep - that always catches me out when I am rolling out these changes. Thanks!